simphony_osp/ontology/entity.py

```diff
@@ -188,7 +201,7 @@ def __str__(self) -> str:
         return (
             f"{self.label}"
             if hasattr(self, "label") and self.label is not None
-            else f"{self.uid}"
+            else f"{self._uid}"
```

Comment:
Why not use the property here (and in the other cases)?

If at some point you extend the getter code (e.g. to run some checks before returning the value), they would not be applied here

Reply:
Done